documentation/modules/auxiliary/scanner/ip/ipidseq.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
```

Comment:
Please use `tools/dev/msftidy_docs.rb` on your file to correct some outstanding issues.